### PR TITLE
Restructure repo, package it, release version 3, cleanup README

### PR DESCRIPTION
**Pull request by Ladsgroup**
Also proper .gitignore

Structure is based on python standards:
https://stackoverflow.com/questions/193161/what-is-the-best-project-structure-for-a-python-application

Bug: T179669

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
 ```
 
-[![Build Status](https://travis-ci.org/Wikidata-lib/PropertySuggester-Python.png?branch=master)](https://travis-ci.org/Wikidata-lib/PropertySuggester-Python)
-[![Coverage Status](https://coveralls.io/repos/Wikidata-lib/PropertySuggester-Python/badge.png?branch=master)](https://coveralls.io/r/Wikidata-lib/PropertySuggester-Python)
-
-
 ## Release Notes
 
+### 3.0.0
+* Restructure repository
```
Comment (thiemowmde) [inline README.md]: Can you please be more specific at what "restructure" means? Otherwise it's quite pointless to mention it at all.

**Review by thiemowmde — APPROVED** [review]
All `__init__.py` appear to be empty … somethings. What are they for? Why not delete them?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,58 @@
-*.pyc
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*~
+ipython/.ipynb_checkpoints
+
+# C extensions
```
[the remark this answers is not in the document]
Comment (Ladsgroup) [reply .gitignore]: It's the default thing that github gives out, it definitely doesn't hurt to have and it's not part of the codebase

File: README.md (hunk 1 above)
Comment (Ladsgroup) [reply README.md]: By restructure I mean moving the files around to live in the proper namespace.

**Comment by Ladsgroup** [comment]
In python __init__.py files are needed, it's PSR-4 of python: https://stackoverflow.com/questions/448271/what-is-init-py-for

**Comment by thiemowmde** [comment]
I disagree that files in the root directory are "not part of the codebase". But I understand it doesn't hurt that much to have such a huge .gitignore file. Feel free to merge.

**Pull request merged**